internal/cli/hud.go

```diff
@@ -0,0 +1,45 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/spf13/cobra"
+	"github.com/windmilleng/tilt/internal/tracer"
+)
+
+type hudCmd struct {
+	traceTags string
+}
+
+func (c *hudCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "hud",
+		Short: "display Tilt/k8s status at a glance",
+	}
+
+	cmd.Flags().StringVar(&c.traceTags, "traceTags", "", "tags to add to spans for easy querying, of the form: key1=val1,key2=val2")
```

Comment:
can we cut this since we're logging the trace id?

Reply:
I've found it useful because it's more memorable/human-readable, i'd say leave it unless it starts being actively annoying